docker-compose.yml

```diff
@@ -49,6 +49,11 @@ services:
     command: ["./container_init.sh"]
     environment:
       - DEPLOYMENT_STAGE=dev
+      - AWS_REGION=us-west-2
+      - AWS_DEFAULT_REGION=us-west-2
+      - AWS_ACCESS_KEY_ID=dev_access_key_id
+      - AWS_SECRET_ACCESS_KEY=dev_secret_access_key
+      - BOTO_ENDPOINT_URL=http://localstack:4566
```

Comment:
For readability, would `AWS_SECREMT_MANAGER_URL` work better? I feel `BOTO` is a Python library that implements AWS Secret Manager and in JS world it's just AWS SDK, so using `BOTO_ENDPOINT_URL` in JS world doesn't convey meaning as much as `AWS_SECREMT_MANAGER_URL`? What do you think?

Reply:
I'm probably going to be leaving it, so we don't accidentally forget it in the case grep'ing for BOTO_ENDPOINT.  We could also rename it so it's named after what its value represents and not where its consumed

Reply:
Ah I see! Yeah sounds like if we change it, it'll have to be across all languages and systems. SGTM!